### PR TITLE
2024 esri copyright

### PR DESCRIPTION
**Pull request by Kizito007**
Missing Esri License File included in source code
#225

### CODE REVIEW

**Comment by gavinr-maps** [comment]
@Kizito007 this is not the correct license text. The Leaflet repos are released under the apache license (you can see it for this repo here: https://github.com/Esri/esri-leaflet-vector/blob/c37e12949d039b0c1a4bf381b8a17e9406d59fec/package.json#L79 ) ... so you can upload the Apache 2 license. You can just get it from the Esri Leaflet repo: https://github.com/Esri/esri-leaflet/blob/master/LICENSE

**Review by gavinr-maps — CHANGES_REQUESTED** [review]
Use correct license text

**Comment by Kizito007** [comment]
Alright @gavinr-maps 
Thanks for the correction. Will make changes ASAP

**Comment by Kizito007** [comment]
License has been corrected to Apache 2.0

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright (c) 2014-2019 Environmental Systems Research Institute, Inc.
```
Comment (gavinr-maps) [inline LICENSE:190]: I don't think having the year updated is totally required, but if you can please update "2019" to the current year I think that would be a bit more correct.

Comment (gavinr-maps) [inline LICENSE:190]:
```suggestion
   Copyright (c) 2014-2024 Environmental Systems Research Institute, Inc.
```

I don't think having the year updated is totally required, but if you can please update "2019" to the current year I think that would be a bit more correct.

**Comment by gavinr-maps** [comment]
thanks!

**Pull request merged**